railties/lib/rails/application/finisher.rb

```diff
@@ -33,6 +33,14 @@ module Finisher
         end
       end
 
+      # Setup default session store if not already set in config/application.rb
+      initializer :setup_default_session_store, before: :build_middleware_stack do |app|
+        unless app.config.session_store
```

Comment:
This introduces one issue. If user has set the `session_store` to `:disabled` then this change overwrites it.